android/src/toga_android/widgets/textinput.py

```diff
@@ -120,6 +108,21 @@ def clear_error(self):
     def is_valid(self):
         return self.native.getError() is None
 
+    def _on_change(self):
+        self.interface.on_change(None)
+        validate = getattr(self.interface, "_validate", None)
+        if validate:
+            validate()
```

Comment:
This seems like it should be managed by subclassing, rather than probing attribute access.